xio-core/src/main/java/com/xjeffrose/xio/http/Message.java

```diff
@@ -0,0 +1,14 @@
+package com.xjeffrose.xio.http;
+
+/**
+ * Interface representing a HTTP1/2 Message (Request/Response) Message: A complete sequence of
+ * frames that map to a logical request or response message. See: <a href= ></a>
```

Comment:
link to https://xjdr.github.io/xio/http.html and we'll put something there to explain our messaging abstraction.